File: files/js/pontuacao.js
if (typeof kotlin === 'undefined') {
  throw new Error("Error loading module 'pontuacao'. Its dependency 'kotlin' was not found. Please, check whether 'kotlin' is loaded prior to 'pontuacao'.");
}
var pontuacao = function (_, Kotlin) {
  'use strict';
  var toString = Kotlin.toString;
  var ensureNotNull = Kotlin.ensureNotNull;
  var Unit = Kotlin.kotlin.Unit;
  function main$lambda(closure$xhttp) {
    return function () {
      var areaDePontos = document.getElementById('scores');
      ensureNotNull(areaDePontos).innerHTML = '<h1>Scores<\/h1>' + toString(closure$xhttp.v.responseText.replaceAll(';', ''));
      return Unit;
    };
  }
  function main() {
    var xhttp = {v: new XMLHttpRequest()};
    xhttp.v.open('GET', 'http://0.0.0.0:8080/dados.txt', true);
    xhttp.v.onreadystatechange = main$lambda(xhttp);
    xhttp.v.send();
  }
  _.main = main;
  main();
  Kotlin.defineModule('pontuacao', _);
  return _;
}(typeof pontuacao === 'undefined' ? {} : pontuacao, kotlin);

File: pontuacao.kt
import org.w3c.dom.*
import kotlinx.browser.*
import org.w3c.xhr.XMLHttpRequest

fun main() {
    var xhttp :dynamic = XMLHttpRequest();
 	xhttp.open("GET", "http://0.0.0.0:8080/dados.txt", true)
 	xhttp.onreadystatechange = {
        val areaDePontos = document.getElementById("scores")
        areaDePontos!!.innerHTML = "<h1>Scores</h1>" + xhttp.responseText.replaceAll(";", "")
    }

 	xhttp.send()
}